### ISSUE TITLE
Some rules/options aren't being picked up when configured in a global AnalyzerConfig 

### ISSUE THREAD

**Issue by xt0rted** [JosefPihrt/Roslynator#884]
**Product and Version Used**: 

- `Roslynator.Analyzers`: 4.0.2
- `Roslynator.CodeAnalysis.Analyzers`: 4.0.2
- `Roslynator.Formatting.Analyzers`: 4.0.2

**Steps to Reproduce**:

I'm trying to put all of my Roslynator settings in a [global AnalyzerConfig](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-files#global-analyzerconfig) so it's easier to share/update between projects.

I enabled `RCS0001` and it reports errors. I then enabled `RCS0011` but it doesn't report errors, and it's option `roslynator_blank_line_between_single_line_accessors` doesn't do anything.

If I move the settings for `RCS0011` to the root `.editorconfig` then the analyzer lights up and reports an error, but not when it's configured in my global config.

The setup I'm using is as follows:

**/Directory.Build.targets**

```xml
<Project>

  <Target Name="AddGlobalCodeAnalysisRules" BeforeTargets="CoreCompile">
    <ItemGroup>
      <EditorConfigFiles Include="$(MSBuildThisFileDirectory)analysis\*.editorconfig" />
    </ItemGroup>
  </Target>

</Project>
```

**/analysis/roslynator.editorconfig**

```.editorconfig
is_global = true

# RCS0001: Add blank line after embedded statement
dotnet_diagnostic.RCS0001.severity = error

# RCS0011: Add/remove blank line between single-line accessors
dotnet_diagnostic.RCS0011.severity = error
roslynator_blank_line_between_single_line_accessors = false
```
